src/app/component/user/components/profile/achievements-friends/achievements-friends.component.ts

```diff
@@ -1,5 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { PROFILE_IMAGES } from '../../../../../../assets/img/profile/profile-images';
```

Comment:
use alias for those paths with many `../../../`

Reply:
Fixed